submariner/multiple-active-gateways.md

```diff
@@ -0,0 +1,100 @@
+# Multiple Active Gateways
+
+## Summary
+
+NOTE: The design details are still a work in progress, so this is currently a draft proposal.
+
+Submariner currently only allows a single gateway to be active at any one time in a given cluster.
+This enhancement proposes that there be an option to allow more than one gateway to be active in
```

Comment:
In this new design, will we be using the same submariner gateway label to mark a node[/s] as Gateway nodes?

And all the labelled Gateway nodes will automatically be part of Active/Active gateways?